assets/js/custom/Translation.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env jquery */
+
+// eslint-disable-next-line no-unused-vars
+function Translation (hasDescription, hasCredit) {
+  const elementsToTranslate = [document.getElementById('name')]
+
+  if (hasDescription) {
+    elementsToTranslate.push(document.getElementById('description'))
+  }
+  if (hasCredit) {
+    elementsToTranslate.push(document.getElementById('credits'))
+  }
+
+  translation(
+    document.getElementById('translate-program'),
+    elementsToTranslate,
+    document.documentElement.lang
+  )
+
+  $('.comment-translate-button').each(function () {
+    const commentId = $(this).attr('id').substring('comment-translate-button-'.length)
+    translation(this, document.getElementById('comment-text-' + commentId), document.documentElement.lang)
+  })
+
+  function translation (buttonElement, textElements, targetLang) {
+    let text = ''
+    if (Array.isArray(textElements)) {
+      const array = []
+      for (let i = 0; i < textElements.length; i++) {
+        array.push(textElements[i].innerText)
+      }
+
+      text = array.join('\n\n')

```

Comment:
We thought it might not be clear which part of the translation is the title or the description or credit. So currently we separate them with 2 new linew.

Reply:
I'm wondering: Is it even possible to understand that we are translating a description/credits when we are clicking a button next to the title?